src/gui_common/CustomRichTextLabel.cs

```diff
@@ -632,4 +644,58 @@ private void OnMetaClicked(object meta)
             }
         }
     }
+
+    private void OnMetaHoverStarted(object meta)
+    {
+        if (!EnableTooltipsForMetaTags || speciesPreviewTooltipScene == null)
+            return;
+
+        if (meta is string metaString && metaString.StartsWith("species", StringComparison.Ordinal))
+        {
+            var speciesCode = GUICommon.ConvertFromSafeMetaTag(metaString.Split("species:")[1]);
+            var species = ThriveJsonConverter.Instance.DeserializeObject<Species>(speciesCode);
+
+            if (species is not MicrobeSpecies)
+                return;
+
+            var tooltip = ToolTipManager.Instance
+                .GetToolTipIfExists<SpeciesPreviewTooltip>(species.FormattedName, "speciesPreview");
+            if (tooltip == null)
+            {
+                tooltip = speciesPreviewTooltipScene.Instance<SpeciesPreviewTooltip>();
+                tooltip.Name = species.FormattedName;
+                ToolTipManager.Instance.AddToolTip(tooltip, "speciesPreview");
```

Comment:
I had to fudge the tooltip system due to the way Godot's meta tags work, since I needed to connect from specific meta tags instead of the whole control, and connect unusual signals.

Reply:
There should be a remove tooltip call to go with the add tooltip call because otherwise the tooltip internal data is never released.